grammar.js

```diff
@@ -1543,7 +1561,7 @@ module.exports = grammar({
     ),
 
     escape_sequence: $ => choice(
-      /\\[\\"'ntbr ]/,
+      /\\./,
```

Comment:
Escape sequences in strings in comments should be more relaxed. It's easier to be relaxed everywhere.